src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -174,6 +174,11 @@ public function getLastResponse(): Response
 
     private function request(RequestInterface $request): Response
     {
+        try {
+            $this->client->setServerParameter('HTTP_HOST', $this->sharedStorage->get('hostname'));
+        } catch (\InvalidArgumentException $exception) {
+        }
```

Comment:
What do you think of that ?
```suggestion
        if ($this->sharedStorage->has('hostname')) {
            $this->client->setServerParameter('HTTP_HOST', $this->sharedStorage->get('hostname'));
        }
```